klippy/extras/axis_twist_compensation.py

```diff
@@ -0,0 +1,306 @@
+# Axis Twist Compensation
+#
+# Copyright (C) 2022  Jeremy Tan <jeremytkw98@gmail.com>
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+
+"""
+[axis_twist_compensation]
+horizontal_move_z: 5
+speed: 50
+calibrate_start_x: 0 ; nozzle's x coordinate at the start of the calibration ! required
+calibrate_end_x: 200 ; nozzle's x coordinate at the end of the calibration ! required
+calibrate_y: 100 ; nozzle's y coordinate during the calibration ! required
+compensation_type: multilinear
+"""
```

Comment:
I can't find such a section in other modules, should I remove it from here?

Reply:
Up to you.  I understood this section to be code comments.  If you feel it isn't helpful it's fine to remove it.

Cheers,
-Kevin